### PR TITLE
{bio}[foss/2020b] MIRA v5.0rc2

### PR DESCRIPTION
**Pull request by DirectorSloan**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MIRA/MIRA-5.0rc2-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+configopts += '--with-tcmalloc-dir=$EBROOTGPERFTOOLS/lib '
+
+builddependencies = [
+    ('flex', '2.6.4'),  
+]
+dependencies = [
+    ('Boost', '1.74.0'),
+    ('zlib', '1.2.11'),
+    ('expat', '2.2.9'),
+]
```
Comment (jfgrimm) [inline easybuild/easyconfigs/m/MIRA/MIRA-5.0rc2-foss-2020b.eb:29]:
```suggestion
    ('gperftools', '2.9.1'),
]
```

**Comment by DirectorSloan** [comment]
Can I provide anything to help with this Pull Request?

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
@DirectorSloan welcome to EasyBuild, and thanks for the contribution.

**Comment by verdurin** [comment]
Going in, thanks @DirectorSloan!

**Pull request merged**